dot/rpc/modules/state.go

```diff
@@ -181,11 +181,13 @@ func (sm *StateModule) GetPairs(_ *http.Request, req *StatePairRequest, res *Sta
 		for k, v := range pairs {
 			*res = append(*res, []string{common.BytesToHex([]byte(k)), common.BytesToHex(v)})
 		}
-
 		return nil
 	}
 
-	reqBytes, _ := common.HexToBytes(*req.Prefix)
+	reqBytes, err := common.HexToBytes(*req.Prefix)
+	if err != nil {
+		return err
```

Comment:
A bit out of scope, but we should return a context wrapping the error

```suggestion
		return fmt.Errorf("cannot convert hex prefix %s to bytes: %w", *req.Prefix, err)
```

Reply:
Reminder @jimjbrettj  :wink: